src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -5,7 +5,23 @@
      * Реализуйте метод `getFibonacciSequence(int n)` который вовзращает массив состоящий
      * из `n`чисел  последовательности Фибоначчи.
      */
-    public int[] getFibonacciSequence(int n) {
-        return null;
+    public static int[] getFibonacciSequence(int n) {
+        int[] arrayOfNumbersFibonacci = new int[n];
+        if (n == 1) {
+            return arrayOfNumbersFibonacci;
+        }
+        if (n == 2) {
+            arrayOfNumbersFibonacci[0] = 0;
```

Comment:
@TarasHryts For if (n == 1) you have implemented return) what about for if (n == 2) ??
I am not adding similar comments to each part. You should check your code more widely then my exactly that place where I have added the comment